setup.py

```diff
@@ -34,7 +34,7 @@
 except pkg_resources.DistributionNotFound:
     pass  # yay!
 
-VERSION = "3.6.dev0"
+VERSION = "3.6.dev1"
```

Comment:
Since we released 3.6 shouldn't it be `3.7.dev0` ?

Reply:
wow. Yes. Thank you.